src/schema/basic_slots.yaml

```diff
@@ -550,7 +550,7 @@ slots:
     multivalued: true
     description: >-
       A list of alternative identifiers for the entity.
-    pattern: '^[a-zA-Z0-9][a-zA-Z0-9_\.]+:[a-zA-Z0-9_][a-zA-Z0-9_\-\/\.,]*$'
+    pattern: '^[a-zA-Z0-9][a-zA-Z0-9_\.]+:[a-zA-Z0-9_][a-zA-Z0-9_\-\/\.,\(\)\=\#]*$'
```

Comment:
what are some examples of alternative identifiers using those additional characters?